resources/metadata/versioning.yaml

```diff
@@ -202,6 +203,13 @@ spec:
       type: jenkins
     - name: gitSshKeyCredentialsId
       type: jenkins
+      aliases:
+        - name: gitCredentialsId
+          deprecated: true
+  containers:
+    - name: mvn
+      image: maven:3.6-jdk-8
```

Comment:
What do we need that container for?

Reply:
When I did not have that I got an error saying mvn executable file not found in $PATH. @stippi2 suggested to add the maven container here and then it worked.